tidy/clang_tidy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
you may want to move these files to the scripts folder and look at using/updating the cmake targets that do some of this already

Reply:
👍